tests/RA_UnitTestHelpers.h

```diff
@@ -35,6 +39,11 @@ template<> static std::wstring ToString<Condition::ConditionType>(const Conditio
     return ra::Widen(CONDITIONTYPE_STR[(int)t]);
 }
 
+template<> static std::wstring ToString<ra::services::Http::StatusCode>(const ra::services::Http::StatusCode& t)
```

Comment:
I think making template specializations `static` is non-standard.

Reply:
Well, I've never heard that before. Since Windows APIs before WinRT are basically non-standard it depends on "undefined" behavior. Meaning that it's undefined in C++ but defined on Windows. If we really do need them to be static we can try suppressing the warnings.

Reply:
Here's the template methods we're specializing. These methods are not contained within a class.
```
	// User should provide a specialization of below ToString() template to use equality asserts in test code.
	// std::wstring ToString(const Q& q) is used for converting type Q in  user readable std::wstring value. 
	// Converted value is emitted in Assert failure message. Specialization for basic types has been provided below.
	template <typename Q> static std::wstring ToString(const Q& q) { static_assert(false, "Test writer must define specialization of ToString<const Q& q> for your class " __FUNCSIG__ "."); }
	template <typename Q> static std::wstring ToString(const Q* q) { static_assert(false, "Test writer must define specialization of ToString<const Q* q> for your class " __FUNCSIG__ "."); }
	template <typename Q> static std::wstring ToString(Q* q)       { static_assert(false, "Test writer must define specialization of ToString<Q* q> for your class " __FUNCSIG__ "."); }
```
However, it does compile if I remove the static. I couldn't find any documentation stating which is preferred (or non-standard). What makes you say that it's non-standard?

(I meant to post this a couple days ago, but apparently forgot to hit the button - as this is duplicating pre-existing functions, we can address in a later PR if necessary)

Reply:
If you compile with /W4 it'll say non-standard extension used.

Reply:
Don't worry about it right now, I have it done (/W4 and conformance) in #161 but can't figure out how to resolve one of the problems since you didn't like how I handled it last time.